README.rst

```diff
@@ -72,6 +73,10 @@ Set the Security Group of this instance to allow inbound traffic as shown below.
 
 The Security Group should allow all outbound traffic.
 
+To start a suitable instance, run ``python start-aws.py``.
+Install and configure Python package ``boto`` to access AWS.
```

Comment:
:question: Copy the instructions from the release process on how to configure `boto`.